ppb_vector/vector2.py

```diff
@@ -156,6 +156,9 @@ def __new__(cls, *args, **kwargs):
 
         return self
 
+    def __reduce__(self):
+        return type(self).__new__, (type(self), self.x, self.y)
```

Comment:
Will that prevent subclasses (esp. ones that add attributes) to be pickled correctly?

Reply:
The improved test answers this positively, in the case of subclasses without extra attributes.

I guess it would make sense to punt the with-attributes case to #144.

Reply:
The pickle docs mention higher-level protocols, but they're not actually implemented by the core stuff? not super clear.